cmd/earthly/account_cmds.go

```diff
@@ -457,7 +457,11 @@ func (app *earthlyApp) actionAccountListTokens(cliCtx *cli.Context) error {
 				fmt.Fprintf(w, " *expired*")
 			}
 		}
-
+		lastAccessedAtStr := "\t"
+		if !token.LastAccessedAt.UTC().IsZero() {
```

Comment:
nit: can we rewrite this to be structured like the if/elses above in order to stay consistent.

additionally let's include a specific "Never" rather than an empty string, so it's possible to use tools like `awk`.

e.g.

```
if token.LastAccessedAt.UTC().IsZero() {
    fmt.Fprint(w, "\tNever")
} else {
    fmt.Fprint(w, "\t%s UTC", token.LastAccessedAt.UTC().Format("2006-01-02T15:04"))
}
```

Reply:
good idea!

yea re if/else I was trying to follow Sam's guideline to avoid else :) but I can see how being consistent is preferable in this case